src/linagora.esn.unifiedinbox/views/unified-inbox/subheader.pug

```diff
@@ -8,7 +8,7 @@ block left
     .inbox-selection-aware-subheader.inbox-selecting(ng-switch-when="true")
       button.btn.btn-link.btn-icon(href='', ng-click="ctrl.unselectAllItems()", title="{{ 'Cancel' | translate }}")
         i.mdi.mdi-close
-      span.title.extra-title.ellipsis {{ '%s selected' | translate:ctrl.getSelectedItems().length }}
+      span.title.extra-title.ellipsis {{ '%s selected' | translate:ctrl.selectedItems }}
```

Comment:
Why the old one binds the length of the array but you bind the array?

Reply:
In the translation, we will translate {{ selectedItems.length }}. I checked